packages/phoenix-evals/src/phoenix/evals/utils/__init__.py

```diff
@@ -0,0 +1,171 @@
+import json
```

Comment:
this file seems like a very strange mix of things which really should not be placed in a single file - I'm cool with punting on this but I'm not sure it deserves a directory. It can just be `utils.py`